package.json

```diff
@@ -10,10 +10,11 @@
   },
   "dependencies": {
     "bcryptjs": "2.4.3",
+    "graphql": "^0.13.2",
```

Comment:
why do we need this?

Reply:
`graphql codegen` fails if there is no `graphql`

Reply:
Did you mean to provide `graphql-cli` instead?

Reply:
Nope! `graphql codegen` command fails with `Error: Cannot find module 'graphql'` if `graphql` package is not installed.